flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -386,25 +386,28 @@ private void commitOperation(
       String operatorId,
       long checkpointId) {
     LOG.info(
-        "Committing {} for checkpoint {} to table {} with summary: {}",
+        "Committing {} for checkpoint {} to table {} branch {} with summary: {}",
         description,
         checkpointId,
         table.name(),
+        branch,
         summary);
     snapshotProperties.forEach(operation::set);
     // custom snapshot metadata properties will be overridden if they conflict with internal ones
     // used by the sink.
     operation.set(MAX_COMMITTED_CHECKPOINT_ID, Long.toString(checkpointId));
     operation.set(FLINK_JOB_ID, newFlinkJobId);
     operation.set(OPERATOR_ID, operatorId);
+    operation.toBranch(branch);
 
     long startNano = System.nanoTime();
     operation.commit(); // abort is automatically called if this fails.
     long durationMs = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startNano);
     LOG.info(
-        "Committed {} to table: {}, checkpointId {} in {} ms",
+        "Committed {} to table: {}, branch: {}, checkpointId {} in {} ms",
```

Comment:
@stevenzwu I added this log for branch when backporting, noticed we were missing the branch after the operation actually committed. I'll add this to Flink 1.16 in a separate PR. Let me know your thoughts.

Reply:
I will be fine if you just add it in this PR. 

Can you check the diff output and make sure no diff related to this backport?
```
git diff --no-index  flink/v1.14/flink/src/ flink/v1.16/flink/src
git diff --no-index  flink/v1.15/flink/src/ flink/v1.16/flink/src
```

Reply:
Checked, verified there's no diff related to this backport.